hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -392,6 +397,41 @@ protected void incrementCounter(Statistic objectsRead, long count) {
     //noop: Use RootedOzoneClientAdapterImpl which supports statistics.
   }
 
+  @Override
+  public void setReplication(String key, short replication) throws IOException {
```

Comment:
Can we avoid duplication between the two adapter classes?

Reply:
Done.